web_ui_src/main.css

```diff
@@ -100,6 +100,39 @@ body.pushable>.pusher, body:not(.pushable) {
   }
 }
 
+/* **************************
+ * Menu things
+ *
+ * Fomantic-UI active class for menus conflicts w/ the submenu active class usage.
+ * Thus we define our own `activated` class.  Below is a adjusted copy from the Fomantic-UI project.
+ * See https://github.com/mhthies/smarthomeconnect/pull/91 for details
+ * ***************************/
+
+.ui.menu .ui.dropdown .menu > .activated.item {
+  background: rgba(0, 0, 0, 0.03) !important;
+  font-weight: bold !important;
+  color: rgba(0, 0, 0, 0.95) !important;
+}
+
+.ui.vertical.menu .dropdown.activated.item {
+  box-shadow: none;
+}
+/* --------------
+     Active
+--------------- */
+.ui.menu .activated.item {
+  background: rgba(50, 50, 50, 0.8);
```

Comment:
Made this is a little bit lighter as w/ the values from fomatic it was really hard to see the difference.